src/python/Dockerfile

```diff
@@ -236,15 +236,14 @@ CMD source venv/bin/activate && \
     cd grapl-notebook && \
     jupyter notebook --ip="0.0.0.0"
 
-#
-# Local provision
-################################################################################
+# test
+FROM grapl-notebook AS grapl-notebook-test
 
-# build
-FROM grapl-python-build AS grapl-provision
+# This file will improve once #444 lands
+# primarily to use pre-downloaded tools instead of downloading again
 
-COPY --chown=grapl python/grapl_provision grapl_local_provision
-COPY --chown=grapl --from=grapl-analyzerlib-build /home/grapl/venv venv
+RUN source venv/bin/activate && \
+    pip install nbqa mypy boto3-stubs[essential]
```

Comment:
well, the `grapl_provision` stuff is all gone now, but I'm not sure why we're installing these dependencies with pip instead of copying in the virutalenv like before. Like I said below, I don't believe I actually made this change... not sure why it's in this change set.

Reply:
If you didn't make these changes, do you mind if I return them back to how they look in `main`?

Reply:
hmm actually, this does look necessary because `grapl_provision` is no longer a thing. I guess I did make this change? I don't remember it but who knows. I don't understand what the comment on L240-241 means regarding "#444" so I'm really not sure what to do here. I propose we leave it for now and investigate in a follow-up story, because I'd really rather not go down the rabbit hole of reverse-engineering this now: 
https://github.com/grapl-security/issue-tracker/issues/574